rubocop.yml

```diff
@@ -336,6 +336,41 @@ Naming/HeredocDelimiterCase:
 Naming/HeredocDelimiterNaming:
   Enabled: false
 
+Naming/InclusiveLanguage:
+  Enabled: true
+  CheckIdentifiers: true
+  CheckConstants: true
+  CheckVariables: true
+  # CheckStrings is false in the rubocop defaults
+  CheckStrings: true
+  CheckSymbols: true
+  CheckComments: true
+  CheckFilepaths: true
+  FlaggedTerms:
+    whitelist:
+      Regex: !ruby/regexp '/white[-_\s]?list/'
+      Suggestions:
+        - allowlist
+        - permit
+    blacklist:
+      Regex: !ruby/regexp '/black[-_\s]?list/'
+      Suggestions:
+        - denylist
+        - block
+    master:
+      Suggestions:
+        - primary
+        - leader
+      AllowedRegex:
+        - !ruby/regexp '/master[_\s\.]key/' # Rails master key
+        - 'blob/master/'
+        - 'origin/master'
+    slave:
+      Suggestions:
+        - replica
+        - secondary
+        - follower
```

Comment:
Why aren't these the defaults from RuboCop itself?

Reply:
Got it. I think we should only define the configs we are overriding. If you dump the full config we can assert that the merged result is the expected one during the review (and also make the PR pass CI).